### DISCUSSION TITLE
How do I export multiple multimedia files from EMu?

### DISCUSSION THREAD

**Discussion by reguzik** [fieldmuseum/EMu-Documentation#360] [FAQs] [answered]
How to I export multiple multimedia files from EMu?

**Comment by reguzik** [comment, marked as answer]
To batch export multimedia assets from the Multimedia module in EMu:

1. Be careful not to export more than 2GB at a time
2. Run the "Multimedia Export" report, following the steps outlined here in the ["How to Export Multimedia" doc](https://docs.google.com/document/d/1TGW4iffHOfdqYUJ7qFZy-YjcUxKQWVDYT9Vyieo4er0/)

Related:  To batch download published media from the [Museum Media public portal (NetX)](https://mm.fieldmuseum.org), follow these steps:
- https://github.com/fieldmuseum/dams-netx/discussions/91

**Comment by magpiedin** [comment]
(corresponding updated Q/A: https://github.com/fieldmuseum/EMu-Documentation/discussions/315 )
